src/utils/helpers.coffee

```diff
@@ -64,6 +65,15 @@ exports.excludeFile = (fileName, options) ->
             # Only instrument files that are inside the project.
             excluded = true
 
+        # For each exclude value try to use it as a pattern to exclude files
+        exclude.map (pattern) ->
+            glob.sync pattern,
```

Comment:
I wanted to make this "all glob based", and get rid of the prefix matching [found here](https://github.com/benbria/coffee-coverage/blob/d37d49dd72313e119dfd306494bb0ef7899ea496/src/utils/helpers.coffee#L92-L95).  It's a bit weird that we prefix match glob patterns.

So, I figured I'd take each match returned by glob, add a '/' to the end, and compare against that.  The problem is, this works fine when your exclude list is ['/node_modules'] (since the matched globs will be the same list.)  When your exclude list is ['/node_modules/**/*'], on the other hand, glob returns potentially thousands of strings for us to compare against.  In my largish project, this was the difference between a 9 second run and a 20 second run.

I think I'm just trying to optimize prematurely though.  :P